bin/node/runtime/src/lib.rs

```diff
@@ -1972,7 +1972,27 @@ impl_runtime_apis! {
 
 	impl pallet_nomination_pools_runtime_api::NominationPoolsApi<Block, AccountId, Balance> for Runtime {
 		fn pending_rewards(member_account: AccountId) -> Balance {
-			NominationPools::pending_rewards(member_account).unwrap_or_default()
+			NominationPools::api_pending_rewards(member_account).unwrap_or_default()
+		}
+
+		fn points_to_balance(pool_id: u32) -> Balance {
+			NominationPools::api_points_to_balance(pool_id)
+		}
+
+		fn balance_to_points(pool_id: u32, new_funds: Balance) -> Balance {
+			NominationPools::api_balance_to_points(pool_id, new_funds)
+		}
+	}
+
+	impl pallet_staking_runtime_api::StakingApi<Block, Balance> for Runtime {
+		fn inflation_rate() -> Perquintill {
+			Staking::api_inflation_rate(
+				Perquintill::from_rational(0_500_000_u64, 1_000_000_u64),
```

Comment:
These come across as magic numbers, should they be pulled from somewhere else or refactored as a const?

Reply:
They are pulled from the `REWARD_CURVE` parameters on line 529. 

They are duplicates of those parameters. That build! macro only accepts integer literals also so I am unsure how we could store these values as variables to save repeating them.

Reply:
Yeah might be unavoidable for now -- if we have to go through with this, make sure to document this in both places -- as in, if someone changes line 529, they should know that they have to update it here as well.